kibble/render/render.go

```diff
@@ -97,29 +95,53 @@ func Render(sourcePath string, buildPath string, cfg *models.Config) int {
 	renderer.Initialise()
 
 	initSW.Completed()
-
 	errCount := 0
-
 	renderSW := utils.NewStopwatchLevel("render", logging.NOTICE)
-	for lang, localeObj := range cfg.Languages {
 
-		locale := localeObj.Code
+	// Use data from APIs if translations_api toggle is enabled.
+	// Otherwise, use data from kibble.json.
+	var defaultLanguage string
+	var languageConfigs map[string]models.LanguageConfig
+	var translationFilePath string
+	if site.Toggles["site_translations_api"] {
```

Comment:
Perhaps this is a good place to log it.

```suggestion
	var useApi = site.Toggles["site_translations_api"]
	
	log.Infof("UseTranslationsApi: %t", useApi )	
	
	if useApi {
```